### PR TITLE
add a docker-compose file

### PR DESCRIPTION
**Pull request by tobinquadros**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: '1'
```
Comment (leejones) [inline docker-compose.yml]: Do you need version 1 here? It might be nice to go with version 2 if possible since this is a new project.

Comment (tobinquadros) [inline docker-compose.yml]: You mean set it to `2` out the gate?

Comment (leejones) [inline docker-compose.yml]: Yeah.

Comment (tobinquadros) [inline docker-compose.yml]: Ah, :smile:

_read the docs™_

**Pull request merged**